jsapp/js/editorMixins/editableForm.es6

```diff
@@ -233,8 +233,9 @@ export default assign({
     }
 
     let surveyJSON = surveyToValidJson(this.app.survey);
+    let surveyJSONWithMatrix = koboMatrixParser({source: surveyJSON}).source;
```

Comment:
Please move definition of `surveyJSONWithMatrix` inside od `if` below, as it is not used anywhere else, and it could be confusing in the future why there are two `surveyJSON`s, if they stay next to each other at the same nesting level :)